### PR TITLE
Replace slider

### PR DESCRIPTION
**Pull request by Hackin7**
image

### CODE REVIEW

**Comment by Hackin7** [comment]
Also happened to change the carousel for the featured to GliderJS

Code in default.hbs, can be moved to a partial soon

Need to confirm the design

**Review by wei2912 — COMMENTED** [review]
Code looks fine overall, but please delete all references to OWL carousel (and the respective files). Also, it seems that `package-lock.json` wasn't updated. Do fix the conflict!

Please make further changes based on @BenitaWZX's remarks on aesthetics of footer (regarding distance between title and logos). Otherwise, I think it's good to go - frankly the aesthetics of this won't be great, but so long as it shows up I think it's alright.

**Comment by Hackin7** [comment]
image

Added the padding, and should have removed js and CSS files of owl carousel

Additionally separated the sliders into separate componenets

File: default.hbs [hunk 1]
```diff
@@ -34,6 +34,8 @@
         <div class="site-content">
             {{{body}}}
         </div>
+        {{> our-partners}}
+        {{> featured-by}}
```
Comment (wei2912) [inline default.hbs:38]: These two should be placed just on the homepage (`home.hbs`), not in every page.

**Review by wei2912 — CHANGES_REQUESTED** [review]
No other comments wrt aesthetics, will let @BenitaWZX comment further. Once that's done, I'll review the code.

**Comment by wei2912** [comment]
Merged in https://github.com/AdvisorySG/dawn-advisory-theme/commit/dbc152e78ef4c78ebb611ad2fb338ca883f0bfec.

**Pull request closed without merging**